src/client.js

```diff
@@ -464,35 +487,33 @@ export default class StitchClient {
 
     let { url, fetchArgs } = this._fetchArgs(resource, method, options);
     if (!options.noAuth) {
-      return this.authedId().then(authedId => {
-        if (!this.authedId) {
-          return Promise.reject(new StitchError('Must auth first', ErrUnauthorized));
-        }
+      if (!this.authedId()) {
```

Comment:
[nit] The more appropriate check is `!this.isAuthenticated()`

Reply:
👍